distro/Makefile

```
@@ -37,12 +44,16 @@ get-artifacts: dist
 	# get cli artifacts
 	cd .. && mvn clean package && cd -
 	cp ../target/hazelcast-command-line-*-jar-with-dependencies.jar ${HZ_LIB}/hazelcast-command-line.jar
-	cp src/artifacts/hazelcast-management-center-*.war ${HZ_ARTIFACTS}
+	wget -O ${MC_INSTALL_TAR} http://download.hazelcast.com/management-center/${MC_INSTALL_TAR}
+	tar -zxf ${MC_INSTALL_TAR}
+	rm -rf ${MC_INSTALL_TAR}
```

It's a single file, right? Then there's no need for `-rf`

```suggestion
	rm ${MC_INSTALL_TAR}
```

---

Yes, this one is a single file. 👍